examples/getting_started/run.py

```diff
@@ -31,11 +31,14 @@
     # load config
     config = spear.get_config(user_config_files=[os.path.realpath(os.path.join(os.path.dirname(__file__), "user_config.yaml"))])
 
-    # create Env object
-    if config.SIMULATION_CONTROLLER.AGENT == "SphereAgent":
-        env = spear.Env(config)
-    elif config.SIMULATION_CONTROLLER.AGENT == "VehicleAgent":
-        env = OpenBotEnv(config)
+    # create spear.Instance object
+    sp_instance = spear.Instance(config)
```

Comment:
Don't we need `spear.configure_system(config)` here?

Reply:
Yes we do, missed it.